### PR TITLE
Remove librpma engine as librpma library is no longer maintained

### PR DESCRIPTION
**Pull request by grom72**
Remove librpma engines as the librpma library project has been archived on GitHub.
 

Please confirm that your commit message(s) follow these guidelines:

1. First line is a commit title, a descriptive one-liner for the change
2. Empty second line
3. Commit message body that explains why the change is useful. Break lines that
   aren't something like a URL at 72-74 chars.
4. Empty line
5. Signed-off-by: Real Name <real@email.com>

Reminders:

1. If you modify struct thread_options, also make corresponding changes in
   cconv.c and bump FIO_SERVER_VER in server.h
2. If you change the ioengine interface (hooks, flags, etc), remember to bump
   FIO_IOOPS_VERSION in ioengines.h.

### CODE REVIEW

File: ci/actions-install.sh [hunk 1]
```diff
@@ -54,8 +54,7 @@ DPKGCFG
                 libiscsi-dev
                 libnbd-dev
                 libpmem-dev
-                libpmem2-dev
-                libprotobuf-c-dev
+                libpmemblk-dev
```
Comment (janekmi) [inline ci/actions-install.sh:57]: Restoring `libpmemblk-dev` here seems unnecessary.

File: fio.1 [hunk 2]
```diff
@@ -2431,7 +2431,7 @@ The TCP or UDP port to bind to or connect to. If this is used with
 this will be the starting port number since fio will use a range of
 ports.
 .TP
-.BI (rdma,librpma_*)port \fR=\fPint
+.BI (rdma)port
```
Comment (janekmi) [inline fio.1:2434]: It seems this bit: `\fR=\fPint` still makes sense. Please consider keeping it.

**Review by janekmi — CHANGES_REQUESTED** [review]

**Review by janekmi — APPROVED** [review]

File: ci/actions-install.sh (hunk 1 above)
Comment (janekmi) [reply ci/actions-install.sh]: ✅

File: fio.1 (hunk 2 above)
Comment (janekmi) [reply fio.1]: ✅

**Comment by vincentkfu** [comment]
I have looked through this and see no issues, but we are having a long weekend in the US, so I will wait until Tuesday to merge.

**Pull request merged**